R/generate_autonewsmd.R

```diff
@@ -186,8 +186,15 @@ generate_autonewsmd <- function(self, private) {
       )
       append_list[["full_changes"]] <- paste0(
         "Full set of changes:\ [`", set_changes,
-        "`](", file.path(repo_url, "compare", set_changes), ")"
+        "`](", file.path(repo_url, "compare", set_changes, fsep = "/"), ")"
       )
+      if (grepl(pattern = "^(http(s)?://)?.*gitlab.*\\/", x = repo_url)) {
```

Comment:
@joundso could you please check if this change solves your issue with the gitlab-comparison URL?

Reply:
Seems to work - thanks a lot!

Reply:
Thx for the confirmation!